enh(release): Add Centreon Web 20.10.13 (#1064)
centreon/centreon-documentation: en/releases/centreon-core.md

## Centreon Web

### 20.10.13

> 

Reviewer: ```suggestion

Release date: 'November 10, 2021'

```

Reviewer: added release date

## Centreon Web

### 20.10.13

> Release date: 'November 10, 2021'

Reviewer 2: ```suggestion
Release date: `November 10, 2021`
```

> Release date: `November 10, 2021`

Reviewer: ```suggestion
```

#### Enhancements

- [CEIP] Product Adoption component integration
- [Core] Add Feature Flipping for Resources Status vs Legacy Pages
- [Core] Compatibility PHP 7.4

#### Bug fixes

- [Authentication] Fixed user information retrieval method for OpenId

> - [CLAPI] Fixed recurrent downtimes exported with CLAPI cannot be imported with CLAPI

Reviewer 2: ```suggestion
- [CLAPI] Fixed recurrent downtimes exported with CLAPI that could not be imported with CLAPI
```

> - [Configuration] Avoid blocking configuration generation when user don't have notifications enabled

Reviewer 2: ```suggestion
- [Configuration] Avoid blocking configuration generation when users don't have notifications enabled
```

- [Configuration] Massive changes on hosts reset untouched values

> - [Configuration] Unwanted empty string parameter for LUA output in Broker configuration

Reviewer 2: ```suggestion
- [Configuration] Fixed unwanted empty string parameter for LUA output in Broker configuration
```

> - [Custom Views] Can't share views with LDAP groups

Reviewer 2: ```suggestion
- [Custom Views] Couldn't share views with LDAP groups
```

> - [Discovery] Error message when filling credentials in job as side-effect of a change regarding downtimes

Reviewer: ```suggestion
- [Discovery] Fixed an issue causing error messages when filling credentials in a Host Discovery job
```
